e2e/utils.go

```diff
@@ -38,7 +40,7 @@ const (
 
 	// liberal default times to wait
 	defaultWaitForRegistration = 8 * time.Second
-	defaultWaitForEvent        = 8 * time.Second
+	defaultWaitForEvent        = 15 * time.Second
```

Comment:
This timeout needed to be relaxed because it was too tight even when tests run sequentially